source/documentation/monitoring_apps/metrics.md

```diff
@@ -123,3 +123,84 @@ If you are not receiving any metrics, check the [logs](/monitoring_apps.html#log
 ### More about monitoring
 
 For more information about monitoring apps, see [Monitoring the status of your service](https://www.gov.uk/service-manual/technology/monitoring-the-status-of-your-service) on the Service Manual.
+
+## Backing service metrics
+
+If you use the [PostgreSQL](deploying_services/postgresql/#postgresql), [MySQL](deploying_services/mysql/#mysql) or [Redis](deploying_services/redis/#redis) backing service, you can view metrics on those backing services in the GOV.UK PaaS admin tool.
+
+When you create your backing service instance, you can access its metrics with no additional work. Your backing service plan does not affect which metrics you can view.
+
+You can view metrics up to 60 calendar days before today's date. You can select any date range within this time period.
```

Comment:
I think we've enabled more than 60 days for now - it's ~1 year at the moment. Perhaps this is here as a bit of future proofing though?

Reply:
Jon and I did some digging around in CloudWatch and found that metrics seemed only to go as far back as 60 days. Although we only looked at really long lived DBs like bosh.

Reply:
As per discussion with @richardTowers will change to 1 year